docs/explanation/bases.rst

```diff
@@ -1,68 +1,66 @@
+:relatedlinks: https://ubuntu.com/about/release-cycle, [Ubuntu&#32;image&#32;on&#32;Docker&#32;Hub](https://hub.docker.com/_/ubuntu), [Ubuntu&#32;image&#32;on&#32;Amazon&#32;ECR](https://gallery.ecr.aws/ubuntu/ubuntu), [Scratch&#32;image&#32;on&#32;Docker&#32;Hub](https://hub.docker.com/_/scratch)
+
 .. _bases_explanation:
 
 Bases
 =====
 
-One mandatory key for every rock project is the
-:ref:`base <rockcraft_yaml_base>`.
+Every rock is built from a *base*, which defines the baseline system that the rock's
+contents are layered on. The systems are always major Ubuntu releases. An empty, or
+*bare* base, is also available for lean images.
+
+The base is declared in the project file by the :literalref:`base <rockcraft_yaml_base>`
+key.
 
-This key defines the baseline system upon which the rock's contents shall be
-layered on. Only Ubuntu and ``bare``
-:ref:`bases <rockcraft_yaml_base>` are supported.
 
 Ubuntu bases
 ------------
 
-When setting an Ubuntu
-:ref:`base <rockcraft_yaml_base>` for a rock, Rockcraft will pull the
-corresponding Ubuntu base container image from a container registry. This is
-exactly the same container one would get from places like
-`Docker Hub <docker_hub_>`_ or `Amazon ECR <ecr_>`_.
+When building an Ubuntu base, Rockcraft pulls the corresponding Ubuntu image from a
+container registry. These Ubuntu images are the same as the ones hosted in registries
+like Docker Hub or Amazon Elastic Container Registry.
+
+The built rock contains any software and utilities that are present in the Ubuntu image.
+
+Setting an Ubuntu base for a rock is especially useful when the goal is to build a rock
+that can be used for general-purpose environments, such as a development working space.
+
+
+LTS and interim bases
+---------------------
 
-The resulting rock will therefore also contain any software and utilities that
-are present in the Ubuntu base container images.
+Ubuntu bases are divided into *LTS* and *interim* bases. An LTS base contains an Ubuntu
+LTS release and has a 10-year support window. An interim base contains an interim Ubuntu
+release and is supported for nine months.
+
+Interim bases are less stable than LTS bases because they contain experimental features
+and have shorter lifespans. They are most suitable for container-based testing the
```

Comment:
Not sure we really want to say interim bases are less stable (also... not my experience ;-p). Maybe just point out that interim bases only have a 9 month lifespan and are thus best suited to preparing for the next LTS release?